src/utils.test.ts

```diff
@@ -0,0 +1,28 @@
+import { extractEnvVars } from './utils';
+
+describe('utils tests', () => {
+  describe('when evn vars are less then max size', () => {
+    it('extractEnvVars should not filter anything', () => {
+      process.env = {
+        a: '2',
+      };
+      const env = extractEnvVars();
+      expect(process.env).toEqual(env);
```

Comment:
```suggestion
      expect(env).toEqual(process.env);
```
Same regarding the other test